### DISCUSSION TITLE
[Bug] Create store in backend

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#307] [Help]
Ahoi!

I tried to create a new store via backend by clicking "Create a new store" in the dropdown navigation in the top left profile section. Nothing happens and is resulting in a JS error: `TypeError: this.openModal is not a function`

**Comment by logeshswell** [comment]
@josias-hosang I can confirm that it is a bug at our end. I will create a bug ticket internally for faster resolution.

**Reply by logeshswell** [reply]
@josias-hosang This issue is fixed now. Can you check and confirm?

**Reply by josias-hosang** [reply]
I can confirm, store creation is now working. Thanks for the fix.
